biotope/2018-06-05_biotopeVision.md

```diff
@@ -0,0 +1,20 @@
+# Biotope is a framework which enables you to develop scalabale, platform-agnostic frontend component ecosystems
+
+## It helps developers to:
+- easily develop ui-components following a unified standard
+- keep focus on developing components, the architecture and dev environment is provided by Biotope.
```

Comment:
I like to keep focus, but in order to be focused the environment should be a lightweight wrapper. This wrapper could be exchanged easily by a custom pipeline feeding back into an environment.

Reply:
I don't get the point. What do you want to add/change in this sentence?

Reply:
@jurekbarth ?

Reply:
For me biotope feels like a big monster monolith, with a lot of conventions. That hinders me from developing components. I would suggest: keep focus on developing, an architecture and a lightweight dev environment is provided by Biotope.

Reply:
@jurekbarth which part of biotope feels like that and what exactly hinders your component development? to know that helps us improve this points. can you please provide that in a seperate document? for the vision: am i write that you just want to remove the word component from the sentence? biotope is all about developing an ecosystem of ui components so i think "keep focus on developing components" is correct. what do you think?

Reply:
You‘re correct biotope is built, for developing ui components. For me it‘s more about what Biotope really provides in order to make me focus on developing these components. I would like to see a more flexible composable Biotope and not that big out of the box doing everything frameworky thing. I would like to be able to add things on the fly with ease. A pluggable system. There could be prebuild pipelines for different things, like Typescript. All in all i would like to be in control of what sugar on top i want. I can not focus on ui component development when i need to figure out how it is supposed to work. Hope that makes it a little bit clearer 👍